packages/react-native-otlp/ios/RNEmbraceOTLP/InternalOpenTelemetry/OpenTelemetryProtocolCommon/common/CommonAdapter.swift

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
for files that were copied can you add a comment with a perma link to their locations in a github repo?

Reply:
done